lib/ProcessContainerFork.js

```diff
@@ -1,17 +1,33 @@
-/**
+  /**
  * Copyright 2013 the PM2 project authors. All rights reserved.
  * Use of this source code is governed by a license that
  * can be found in the LICENSE file.
  */
 // Inject custom modules
 if (process.env.pmx !== 'false') {
-  require('pmx').init({
+  const pmx = require('pmx');
+  pmx.init({
     transactions: (process.env.km_link === 'true' && (process.env.trace === 'true' || process.env.deep_monitoring === 'true')) || false,
     http: process.env.km_link === 'true' || false,
     v8: process.env.v8 === 'true' || process.env.deep_monitoring === 'true' || false,
     event_loop_dump: process.env.event_loop_inspector === 'true' || process.env.deep_monitoring === 'true' || false,
     deep_metrics: process.env.deep_monitoring === 'true' || false
   });
+
+  if(require('semver').satisfies(process.versions.node, '>= 8.0.0')) {
```

Comment:
We should also add the custom action in the `ProcessContainer` which is used in the cluster mode